cpp/include/cudf/strings/find_multiple.hpp

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+namespace strings
+{
+
+/**
+ * @brief Returns a column with position values where each
+ * of the target strings are found in each string.
+ *
+ * If the string is not found, -1 is returned for that entry.
```

Comment:
What is `the string`? There are multiple in `targets`, right?  Let's clarify:

`output[i] contains the index of `strings[i]` in `targets`, or -1 if `strings[i]` is not contained in `targets`.